### PR TITLE
[service.mailnotifier] 0.2.9

### PR DESCRIPTION
**Pull request by senufo**
mailnotifier is a service that allows you to see the arrival of new mail on the HOME page of kodi

### CODE REVIEW

File: service.mailnotifiers/Home_Confluence_mail.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+	<control type="image">
```
Comment (phil65) [inline service.mailnotifiers/Home_Confluence_mail.xml]: This file looks completely wrong.

File: service.mailnotifiers/addon.xml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+     <import addon="xbmc.python" version="2.19.0"/>
+  </requires>
+  <extension point="xbmc.service"
+             library="notifier.py" start="login">
+  </extension>
+ <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <summary lang="en">Email Notifier</summary>
+    <summary lang="fr">Notification de l'arrivée des mails</summary>
+    <description lang="en">Notify new email[CR]Please note there is a bug with Aeon MQ5, messages superimposed. I have not yet found a solution[CR]
```
Comment (phil65) [inline service.mailnotifiers/addon.xml]: This should be a description, not some setting explanations.

File: service.mailnotifiers/notifier.py [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+    SEP = '| '
+# Control ID in window Home.xml
+MsgBox = None
+MsgBoxId = None
+
+start_time = 0
+# Flag for add or not control in HOME
+re_added_control = False
+# Verify if kodi work
+while (not xbmc.abortRequested):
```
Comment (phil65) [inline service.mailnotifiers/notifier.py]: Please use xbmc.Monitor class (see http://kodi.wiki/view/Service_add-ons )

File: service.mailnotifiers/resources/language/English/strings.xml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (phil65) [inline service.mailnotifiers/resources/language/English/strings.xml]: Please convert your language files .po format.

**Comment by senufo** [comment]
I made the corrections. What is the best way.  I have to update this commit or make a new commit in a new PR ?

**Comment by phil65** [comment]
update this commit (by force-pushing to same branch)

**Comment by learningit** [comment]
You also need to squash the commits down to a single commit. See https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit for details.

**Comment by MartijnKaijser** [comment]
Sorry but this is the wrong repo.
You addon is a script/service so it belongs here https://github.com/xbmc/repo-scripts

**Pull request closed without merging**